src/Dto/Field.cs

```diff
@@ -1,11 +1,129 @@
+        /// Indicates the values should be displayed and/or edited with a single-line text box.
+        /// </summary>
+        SingleLine,
+
+        /// <summary>
+        /// Indicates the values should be displayed and/or edited with a multi-line text box.
+        /// </summary>
+        MultiLine
+    }
+
     public partial class Field
     {
-        /// <summary>The display name for the field</summary>
-        [Newtonsoft.Json.JsonProperty( "discreteValues", Required = Newtonsoft.Json.Required.Default, NullValueHandling = Newtonsoft.Json.NullValueHandling.Ignore )]
+        /// <summary>
+        /// The possible discrete values for the field, if <see cref="Type"/> is <see cref="FieldType.Discrete"/>.
+        /// </summary>
+        [JsonProperty( "discreteValues", Required = Required.Default, NullValueHandling = NullValueHandling.Ignore )]
         public IDictionary<long, string> DiscreteValues { get; set; }
+
+        /// <summary>
+        /// The number edit style for the field, if <see cref="Type"/> is <see cref="FieldType.Number"/>.
+        /// </summary>
+        [JsonProperty( "numberEditStyle", Required = Required.Default, NullValueHandling = NullValueHandling.Ignore )]
+        public string NumberEditStyle { get; set; }
+
+        /// <summary>
+        /// The number interpretation for the field, if <see cref="Type"/> is <see cref="FieldType.Number"/>.
+        /// </summary>
+        [JsonProperty( "numberInterpretation", Required = Required.Default, NullValueHandling = NullValueHandling.Ignore )]
+        [JsonConverter( typeof( StringEnumConverter ) )]
+        public NumberInterpretation? NumberInterpretation { get; set; }
+
+        /// <summary>
+        /// The possible minimum value for the field, if <see cref="Type"/> is <see cref="FieldType.Number"/>.
+        /// </summary>
+        [JsonProperty( "numberMinValue", Required = Required.Default, NullValueHandling = NullValueHandling.Ignore )]
+        public double? NumberMinValue { get; set; }
+
+        /// <summary>
+        /// The possible maximum value for the field, if <see cref="Type"/> is <see cref="FieldType.Number"/>.
+        /// </summary>
+        [JsonProperty( "numberMaxValue", Required = Required.Default, NullValueHandling = NullValueHandling.Ignore )]
+        public double? NumberMaxValue { get; set; }
+
+        /// <summary>
+        /// The possible number units for the field, if <see cref="Type"/> is <see cref="FieldType.Number"/>.
+        /// </summary>
+        [JsonProperty( "numberUnits", Required = Required.Default, NullValueHandling = NullValueHandling.Ignore )]
+        public string NumberUnits { get; set; }
+
+        /// <summary>
+        /// True if a date field is in local time, if <see cref="Type"/> is <see cref="FieldType.DateTime"/>.
+        /// </summary>
+        [JsonProperty( "dateTimeLocal", Required = Required.Default, NullValueHandling = NullValueHandling.Ignore )]
+        public bool? DateTimeLocal { get; set; }
```

Comment:
This one confuses me a bit in relation to DateTimeType - since the latter's enum values all reference UTC (e.g. DateTimeUtc, DateOnlyUtc, etc.)

Reply:
I think in reality this is only ever false, and it's confusing that the API returns it at all, but I chose to pass through what the API was doing here